common/src/main/java/com/tencent/rss/common/config/RssBaseConf.java

```diff
@@ -110,7 +110,7 @@
   public static final ConfigOption<Integer> RPC_MESSAGE_MAX_SIZE = ConfigOptions
       .key("rss.rpc.message.max.size")
       .intType()
-      .defaultValue(Integer.MAX_VALUE)
+      .defaultValue(1073741824)
```

Comment:
1073741824 -> 1024 * 1024  * 1024 to make it readable.

Reply:
OK